pkgs/by-name/as/ast-grep/package.nix

```diff
@@ -26,12 +28,22 @@ rustPlatform.buildRustPackage rec {
     rm .cargo/config.toml
   '';
 
-  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
-    installShellCompletion --cmd sg \
-      --bash <($out/bin/sg completions bash) \
-      --fish <($out/bin/sg completions fish) \
-      --zsh <($out/bin/sg completions zsh)
-  '';
+  cargoBuildFlags = [
+    "--package ast-grep"
+    "--bin ast-grep"
+  ];
```

Comment:
Is this nescesarry? If not, it's just added fragility.

Reply:
This is nescesarry to remove `sg` command which has conflict with linux's builtin sg command.
https://man7.org/linux/man-pages/man1/sg.1.html

Reply:
https://ast-grep.github.io/guide/quick-start.html

In ast-grep's doc, it also remove `sg` on Linux